src/hyp3_autorift/process.py

```diff
@@ -319,6 +331,45 @@ def apply_landsat_filtering(reference_path: str, secondary_path: str) \
     return reference_path, reference_zero_path, secondary_path, secondary_zero_path
 
 
+def get_lat_lon_from_ncfile(ncfile: Path) -> Tuple[float, float]:
+    with Dataset(ncfile) as ds:
+        var = ds.variables['img_pair_info']
+        return var.latitude, var.longitude
+
+
+def point_to_prefix(platform_shortname: str, lat: float, lon: float) -> str:
+    """
+    Returns a string (for example, N78W124) for directory name based on
+    granule centerpoint lat,lon
+    """
+    NShemi_str = 'N' if lat >= 0.0 else 'S'
+    EWhemi_str = 'E' if lon >= 0.0 else 'W'
+
+    outlat = int(10*np.trunc(np.abs(lat/10.0)))
+    if outlat == 90:  # if you are exactly at a pole, put in lat = 80 bin
+        outlat = 80
+
+    outlon = int(10*np.trunc(np.abs(lon/10.0)))
+
+    if outlon >= 180:  # if you are at the dateline, back off to the 170 bin
+        outlon = 170
+
+    return f'{NShemi_str}{outlat:02d}{EWhemi_str}{outlon:03d}'
+
+
+def get_opendata_prefix(file: Path):
+    # filenames have form GRANULE1_X_GRANULE2
+    scene = file.name.split('_X_')[0]
+
+    platform_shortname = get_platform(scene)
+    lat, lon = get_lat_lon_from_ncfile(file)
+    lat_lon_prefix_component = point_to_prefix(platform_shortname, lat, lon)
+
+    dir_path = f'velocity_image_pair/{PLATFORM_SHORTNAME_LONGNAME_MAPPING[platform_shortname]}/v02'
+    prefix = os.path.join(dir_path, lat_lon_prefix_component)
```

Comment:
I'd still prefer all of this logic in `point_to_prefix` so it's actually returning the prefix. Otherwise we should change the function name.